XCUITests/HistoryTests.swift

```diff
@@ -48,4 +48,108 @@ class HistoryTests: BaseTestCase {
         waitforExistence(app.tables.cells["HistoryPanel.recentlyClosedCell"])
         XCTAssertFalse(app.tables.cells.staticTexts[webpage["label"]!].exists)
     }
+
+    func testRecentlyClosedOptionAvailable() {
+        navigator.goto(HistoryRecentlyClosed)
+        waitforNoExistence(app.tables["Recently Closed Tabs List"])
+
+        // Go to a web site and check whether the option is enabled
+        navigator.goto(BrowserTab)
+        waitUntilPageLoad()
+        navigator.goto(BrowserTabMenu)
+        navigator.goto(HistoryRecentlyClosed)
+        waitforNoExistence(app.tables["Recently Closed Tabs List"])
+
+        // Now close the web site and check whether the option is enabled
+        navigator.performAction(Action.CloseTab)
+        navigator.goto(BrowserTabMenu)
+        navigator.goto(HistoryRecentlyClosed)
+        waitforExistence(app.tables["Recently Closed Tabs List"])
+        XCTAssertTrue(app.tables.cells.staticTexts["The Book of Mozilla"].exists)
+
+        // This option should be enabled on private mode too
+        navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
+        navigator.goto(HistoryRecentlyClosed)
+        waitforExistence(app.tables["Recently Closed Tabs List"])
+    }
+
+    func testClearRecentlyClosedHistory() {
+        navigator.performAction(Action.CloseTab)
+        navigator.goto(BrowserTabMenu)
+        navigator.goto(HistoryRecentlyClosed)
+        waitforExistence(app.tables["Recently Closed Tabs List"])
+        XCTAssertTrue(app.tables.cells.staticTexts["The Book of Mozilla"].exists)
```

Comment:
Let's put a quick comment on why the "Book of Mozilla" cell should appear, that'll help people to better understand the test.

Reply:
Sure, I will add it. Thanks!